tobago-example/tobago-example-demo/src/main/webapp/menu.xhtml

```diff
@@ -115,7 +115,8 @@
             <tc:link outcome="/content/000-intro/96-logging-info/Logging_Info.xhtml" immediate="true" label="Logging Info"
                      disabled="#{! serverInfo.enabled}"/>
             <tc:separator/>
-            <tc:link link="http://myfaces.apache.org/tobago" label="Tobago Home"/>
+            <tc:link link="https://myfaces.apache.org/#/tobago" label="Tobago Project" image="bi-box-arrow-up-right"/>
```

Comment:
Please use target="_blank" for links that should open a new browser tab.

Reply:
-1
Please don't use target=_blank in the Demo regularly. The user may press the <CRTL> oder <CMD> key by himself.